regulatory/documents/510k.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Same questions about this file: should we change anything in it or is it good as-is?

Reply:
I think this is meant to be a placeholder for the contents of a 510(k) submission.

Let us disregard this for the moment.